frontend/src/app/home/containers/Home.js

```diff
@@ -51,12 +66,69 @@ export default class Home extends React.Component {
 
   }
 
+  sellStock(stock, quantity, price){
+    let owned = 3;
+    if (quantity === ''){
+      alert("Can't sell zero shares!")
+    }
+    else if (quantity > owned){
+      alert("Can't sell more shares than you own!")
+    }
+    else {
+      alert(`Sold ${quantity}x ${stock} shares!`)
+    }
+
+  }
+
+  postFeed() {
+    if (this.state.post !== "") {
+      const jwt = localStorage.getItem("jwt");
+      const url = `https://vm344c.se.rit.edu/api/twitter_proxy/tweet/`;
```

Comment:
Host?

Reply:
`https://vm344c.se.rit.edu` should be omitted when this is merged.

Reply:
so you want a const that's referenced with the endpoint concantenated instead? I don't know what you mean tbh

Reply:
The `proxy` field in the package.json allows inserting the origin (scheme, host, port) for calls made through fetch. We do this because changing that one setting is much easier than changing the origin for every single call when deploying.

Reply:
ok gotcha so just call "proxy.host"?

Reply:
No. The proxy value present in the config file is injected via Webpack at compile-time without any additional work on your end. All you need to be concerned about is the `api/twitter_proxy/tweet` section.

Again this only applies when this is eventually merged, If you are testing against the live VM, then yes, leave what you have currently alone until you prepare to merge.

Reply:
I can't test at all, inside this project at least. I've tested by spoofing data and checking against that....